Jenkinsfile_CNP

```diff
@@ -39,16 +43,63 @@ static LinkedHashMap<String, Object> secret(String secretName, String envVar) {
   ]
 }
 
+// Vars needed for AKS testing
+env.TEST_S2S_NAME = 'bulk_scan_processor_tests'
+env.TEST_S2S_URL = 'http://rpe-service-auth-provider-aat.service.core-compute-aat.internal'
+env.TEST_SCAN_DELAY = '4000'
+env.RESOURCE_GROUP = 'bulk-scan-aks-rg'
```

Comment:
remove rg from the name please, https://tools.hmcts.net/confluence/display/CNP/Naming+Conventions